packages/core/src/Page/index.js

```diff
@@ -468,6 +483,17 @@ class Page {
       ...layoutManager.getLayoutPageNjkAssets(this.layout),
     };
 
+    const VueAppPage = new Vue({
+      template: `<div id="app">${content}</div>`,
+      // template: '<script>  window.location.href = "gettingStarted.html"</script>',
+    });
+
+    content = await renderToString(VueAppPage);
```

Comment:
Thanks for the preview @wxwxwxwx9 

Not so straightforward unfortunately - note you've erased the original template / render function availability for the client (browser) in doing this.
i.e. vue in the browser now dosen't know what your original application is, and is trying to treat the ssr gibberish output as the application (which it isn't). (note it previously / currently does because the output is the **unrendered** app template)

You'll need to compile, and pass in the application to the client as well. (the first part of https://github.com/MarkBind/markbind/issues/1406)

Reply:
Thanks for the pointers! Will work on it :-)